render_client.go

```diff
@@ -108,11 +108,11 @@ func (renderer *Renderer) RenderClient() ([]byte, error) {
 		}
 
 		if method.Method == "POST" {
-			f.WriteLine(`body := new(bytes.Buffer)`)
+			f.WriteLine(`reqBody := new(bytes.Buffer)`)
```

Comment:
Nit: I might like the name `payload` better. It's short/one word (like `body`), but implies it's being sent (e.g. as a request body). WDYT?

Reply:
Done.

As a side note (because I was initially confused as well why I changed it to `reqBody`): [Here](https://github.com/googleapis/gnostic-go-generator/pull/16/files#diff-5d6b92364de5edfd9f7f8caf8fbe11b8R137) we declare a `body` variable again (this time the response body). Therefore it would not generate valid Go code if we would use `body` for the request body as well.